ApplicationPattern.yaml

```diff
@@ -1067,6 +1146,7 @@ paths:
                 service-log-application: 'ExecutionAndTraceLog'
                 service-log-application-release-number: '2.0.0'
                 service-log-operation: '/v1/record-service-request'
+                service-log-protocol: 'HTTP'
```

Comment:
line 1201: callback uri definition for ServiceRequestCausesLoggingRequest has protocol https which needs to be updated to reference service-log-protocol 
{$request.body#service-log-protocol}://...